trieste/models/interfaces.py

```diff
@@ -475,3 +475,49 @@ def conditional_predict_y(
         raise NotImplementedError(
             f"Model {self!r} does not support predicting observations, just the latent function"
         )
+
+
+class EnsembleModel(ProbabilisticModel):
```

Comment:
If my PR goes in first, then this will want to be marked as a Protocol

```suggestion
class EnsembleModel(ProbabilisticMode, Protocoll):
```